spec/takeaway_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'takeaway'
+describe Takeaway do
+  subject { Takeaway.new }
+  let(:dish) { double("Chicken") }
+  let(:price) { double("£4") }
+  let(:choice) { double(:choice) } 
+
+  describe "#initialize" do
+    it "menu array" do
+      expect(subject.menu).to be_truthy
```

Comment:
This test could probably be removed — you would use `be_truthy` to verify a value is "true" (if it'd be a boolean, in something like `expect(user.allowed?).to be_truthy`. Since `subject.menu` returns a hash, it's probably not the right RSpec matcher to express that `subject.menu` should be a hash

Maybe something better would be:
```ruby
expect(subject.menu).to be_a(Hash)
```